event-driven-programming/tkinter/kiaras_cakes.py

```diff
@@ -0,0 +1,142 @@
+# Isaac Computer Science
+# Usage licensed under the Open Government Licence v3.0
+
+from tkinter import *
+from tkinter import messagebox
+import os
+
+USERID = "Kiara01"
+PASSWORD = "doughnut"
+
+
+def display_order_form():
+    """Displays order entry form"""
+
+    def clear_form():
```

Comment:
Do these functions have to be inside of other functions, it will be very unusual to students...